aws/resource_aws_iam_instance_profile.go

```diff
@@ -69,13 +69,18 @@ func resourceAwsIamInstanceProfile() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"tags": tagsSchema(),
+			"tags":     tagsSchema(),
+			"tags_all": tagsSchemaComputed(),
```

Comment:
Ooo i missed on first review @bflad: just need to update this resource's Read method to set these attributes, hidden in the `instanceProfileReadResult` func

Reply:
Updated in https://github.com/hashicorp/terraform-provider-aws/pull/18723/commits/7dc456f1093f7a695dbbb67fa8dcfff3f9712875